cypress/integration/site-search/index.spec.ts

```diff
@@ -0,0 +1,39 @@
+describe('Docs website search', () => {
+  let searchTerm = '';
+  before(() => {
+    // Start an XHR server with the built-in cypress utilities.
+    cy.server();
+    cy.route('**/**/page-data.json').as('pageDataResponse');
+    cy.route('**/**/app-data.json').as('appDataResponse');
+    cy.visit('/');
+    // Wait for the JS assets to load since they are necessary for the search functionality.
+    cy.wait(['@pageDataResponse', '@appDataResponse']);
+  });
+
+  beforeEach(() => {
+    cy.server();
+    cy.route({url: 'https://bh4d9od16a-dsn.algolia.net/1/indexes/*/**', method: 'POST'}).as('searchRequest');
```

Comment:
If you really want to wait for the request it looks like you can wildcard the whole domain, and the alias will still work as expected, which is cool.

`https://**.algolia.net/**`

We're at least a little less dependent on the url structure that way.

Reply:
Ohhh nice! Yeah I was playing around with the route matching with this older version and was struggling a bit to generalize. I'll make that change. Thanks!